src/mqtt/qmqtt_frame.h

```diff
@@ -32,9 +32,13 @@
 #ifndef QMQTT_FRAME_H
 #define QMQTT_FRAME_H
 
-#include <qmqtt_global.h>
+#include "qmqtt_global.h"
```

Comment:
why? the angle brackets in public headers were used intentionally...

Reply:
Intentionally might have been, but not consistently in the sense that then all public headers need to be referenced by the rhombus `<>` in each file, regardless if it is another public/private header or a source file.

Looking at the code **it was a real mess** so that i decided to convert everything into quotes `""`.

Reply:
then plz convert everything into angle brackets in .h files and into quotes in .cpp files, to be on par with official Qt modules

Reply:
Addressed